bin/KG/dictionariesManager..js

```diff
@@ -47,6 +47,8 @@ var DictionariesManager = {
     },
 
     indexSource: function (indexName, data, _options, callback) {
+        if (typeof data === "string") data = JSON.parse(data);
```

Comment:
the data is not expected to be a string, so let's throw an exception if that is not the case, but let's not convert on the fly to accept multiple types.